release-team/role-handbooks/release-notes/README.md

```diff
@@ -0,0 +1,124 @@
+creates a draft in markdown of the identified major themes to send to sig-leads for feedback/confirmation
+- If gentle nudging of sig leads is not effective in retrieving feedback/confirmation, the release-notes team can
+use a reasonable amount of creative liberity in completing the notes
+- A "Known Issues" section will also be created in a github issue to be added to the release notes before release date
+- The confirmed upon notes are cleaned up and copy edited by the release-notes team to ensure uniform language/style
+is used.
+- Kubernetes is released with the completed notes!
+
+## Tools
+
+- [Release notes tool](https://github.com/marpaia/release-notes)
+- [Hackmd](https://hackmd.io/)
+
+
+## Todos
+
+Update this section at the end of each release for the next team
+
+### Post 1.12 todos
+
+- Implement new functionality in release-notes tool to process language in generated release notes file
+
+  Goals:
+    - Generate uniform style across release notes (ie. past tense, formatting)
+    - Decrease copy editing time
+
+## Release Milestone Activities:
+
+<table>
+  <tr>
+    <td>RELEASE WEEK</td>
+    <td>ACTIVITIES</td>
+  </tr>
+  <tr>
+    <td>1</td>
+<td>- Begin running release-notes script for ongoing collection of release notes
+<br>- Begin attending release team meetings
+<br>- Informal intro meeting with release notes team to discuss contact information and logistics </td>
+  </tr>
+  <tr>
+    <td>2</td>
+<td>- Same as above</td>
+  </tr>
+  <tr>
+    <td>3</td>
+    <td>- Begin attending burndown meetings
+    <br>- Same as above</td>
+  </tr>
+  <tr>
+    <td>4</td>
+    <td>- Same as above</td>
+  </tr>
+  <tr>
+    <td>5</td>
+    <td>- Same as above</td>
+  </tr>
+  <tr>
+    <td>6</td>
+    <td>- Same as above</td>
+  </tr>
+  <tr>
+    <td>7</td>
+    <td>- Same as above</td>
+  </tr>
+  <tr>
+    <td>8</td>
+    <td>- Same as above</td>
+  </tr>
+  <tr>
+    <td>9</td>
+    <td>- Former Release Notes lead helps identify new leads and shadows
+    <br>- Create Google Doc for generated release notes and share with release-notes team
+    <br>- Share created doc with release-team
+<br>- Start determining major themes for release notes template to send to sig-leads </td>
+  </tr>
+  <tr>
+    <td>10</td>
+    <td>- Finalize lead and shadow roles for subsequent release
+    <br>- Work with sig-leads to finalize major themes for release-notes
+    <br>- Copy edit notes for spelling, grammar and uniform style</td>
+  </tr>
+  <tr>
+    <td>11</td>
+    <td>- Repeat previous tasks until final copy is complete
+    <br>- Ensure "known issues" section is incorporated into final document</td>
+  </tr>
+  <tr>
+    <td>12</td>
+    <td>- Release day
+<br>- Copy notes from Google Doc to HackMD in markdown
+<br>- Final version of release notes committed for release </td>
+  </tr>
+  <tr>
+    <td>13</td>
+    <td>- Release retrospective participation</td>
+  </tr>
+</table>
```

Comment:
How about making this table to markdown? Like https://github.com/kubernetes/sig-release/pull/287#issuecomment-420788626 And we could merge some weeks with the same tasks.

Reply:
Oh! Of course! Great suggestion :) I just wish I had realized I could do that sooner! :laughing: